lib/nerves_hub_link/socket.ex

```diff
@@ -102,6 +103,7 @@ defmodule NervesHubLink.Socket do
       |> assign(iex_pid: nil)
       |> assign(iex_timer: nil)
       |> assign(uploader_pid: nil)
+      |> assign(upload_path: config.upload_path)
```

Comment:
This might be slightly confusing since there is also an `uploader_pid`, but that is technically completely different. I don't know if I have a better name though.  `data_dir`? `storage_path`? `:downloads`?

Reply:
Good catch. I hate that we're uploading and downloading on both sides lol. `data_dir` sounds like the best choice